src/pages/HomePage.js

```diff
@@ -0,0 +1,37 @@
+import Heading from '../components/Header'
+import PropTypes from 'prop-types'
+import ConcertList from '../pages/SearchPage'
+import ConcertCard from '../components/ConcertCard'
+
+HomePage.propTypes = {
+  pageName: PropTypes.string,
+  onNavigate: PropTypes.func,
+  bookmarks: PropTypes.array,
+}
+
+export default function HomePage({ bookmarks, pageName, onNavigate }) {
+  return (
+    <div>
+      <Heading pageName={pageName} />
+      <ConcertList>
+        {bookmarks.length === 0 ? (
```

Comment:
You could shorten the code by writing it like this: {bookmaks.length > 0 ? instead of "==="